Sources/CodeEditTextView/CodeEditTextView.swift

```diff
@@ -27,7 +27,7 @@ public struct CodeEditTextView: NSViewControllerRepresentable {
         font: Binding<NSFont>,
         tabWidth: Binding<Int>,
         lineHeight: Binding<Double>,
-        overScrollLineCount: Binding<Int> = .constant(0),
+        overScrollRatio: Binding<Double> = .constant(0.0),
```

Comment:
please add `overScrollRatio` to the DocC comment annotation

Reply:
This still needs to be resolved

Reply:
For clarification: I meant the doc comment above the init describing all the parameters.